msgid ""
msgstr "Project-Id-Version: pyload\n"
"Report-Msgid-Bugs-To: 'bugs@pyload.org'\n"
"POT-Creation-Date: 2014-04-08 15:01+0200\n"
"PO-Revision-Date: 2014-04-09 05:38-0400\n"
"Last-Translator: pyloadTeam <team@pyload.org>\n"
"Language-Team: Dutch\n"
"Language: nl_NL\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Generator: crowdin.net\n"

#: module/Core.py:182
msgid "Received Quit signal"
msgstr "Quit signaal ontvangen"

#: module/Core.py:303
#, python-format
msgid "pyLoad already running with pid %s"
msgstr "pyLoad is al actief met pid %s"

#: module/Core.py:317
#, python-format
msgid "Failed changing group: %s"
msgstr "Veranderen van groep mislukt: %s"

#: module/Core.py:327
#, python-format
msgid "Failed changing user: %s"
msgstr "Veranderen van gebruiker mislukt: %s"

#: module/Core.py:329
msgid "folder for logs"
msgstr "map voor logs"

#: module/Core.py:340
msgid "Starting"
msgstr "Startend"

#: module/Core.py:341
#, python-format
msgid "Using home directory: %s"
msgstr "Gebruik hoofdmap: %s"

#: module/Core.py:350
msgid "pycrypto to decode container files"
msgstr "Container bestanden decoderen met pycrypto"

#: module/Core.py:353
msgid "folder for temporary files"
msgstr "map voor tijdelijke bestanden"

#: module/Core.py:358
msgid "folder for downloads"
msgstr "map voor downloads"

#: module/Core.py:361
msgid "OpenSSL for secure connection"
msgstr "OpenSSL voor beveiligde verbinding"

#: module/Core.py:365
msgid "Moving old user config to DB"
msgstr "Oude configuratie naar DB verplaatsen"

#: module/Core.py:368
msgid "Please check your logindata with ./pyload.py -u"
msgstr "Controleer je logingegevens met ./pyload.py -u"

#: module/Core.py:371
msgid "All links removed"
msgstr "Alle links verwijderd"

#: module/Core.py:402
#, python-format
msgid "Downloadtime: %s"
msgstr "Downloadtijd: %s"

#: module/Core.py:412
#, python-format
msgid "Free space: %s"
msgstr "Beschikbare ruimte: %s"

#: module/Core.py:432
msgid "Activating Accounts..."
msgstr "Accounts worden geactiveerd..."

#: module/Core.py:438
msgid "Activating Plugins..."
msgstr "Plugins aan het activeren..."

#: module/Core.py:441
msgid "pyLoad is up and running"
msgstr "pyLoad is lopende"

#: module/Core.py:460
msgid "restarting pyLoad"
msgstr "pyLoad wordt herstart"

#: module/Core.py:464
msgid "pyLoad quits"
msgstr "pyLoad wordt afgesloten"

#: module/Core.py:521
#, python-format
msgid "Install %s"
msgstr "Installeren %s"

#: module/Core.py:557
#, python-format
msgid "could not find %(desc)s: %(name)s"
msgstr "Niet te vinden %(desc)s: %(name)s"

#: module/Core.py:559
#, python-format
msgid "could not create %(desc)s: %(name)s"
msgstr "Niet aan te maken %(desc)s : %(name)s"

#: module/Core.py:580
msgid "shutting down..."
msgstr "afsluiten..."

#: module/Core.py:597
msgid "error while shutting down"
msgstr "fout tijdens afsluiten"

#: module/Core.py:661
msgid "killed pyLoad from Terminal"
msgstr "pyLoad is afgesloten via Terminal"

#: module/database/DatabaseBackend.py:174
msgid "Filedatabase was deleted due to incompatible version."
msgstr "Bestandsdatabase is verwijderd vanwege niet verenigbare versie."

#: module/database/DatabaseBackend.py:189
msgid "Filedatabase could NOT be converted."
msgstr "Bestandsdatabase kan NIET geconverteerd worden."

#: module/database/DatabaseBackend.py:198
msgid "Database was converted from v2 to v3."
msgstr "Database is geconverteerd van v2 naar v3."

#: module/database/DatabaseBackend.py:206
msgid "Database was converted from v3 to v4."
msgstr "Database is geconverteerd van v3 naar v4."

#: module/database/DatabaseBackend.py:252
msgid "Converting old Django DB"
msgstr "Oude Django DB omzetten"

#: module/database/FileDatabase.py:45
msgid "finished"
msgstr "compleet"

#: module/database/FileDatabase.py:45
msgid "offline"
msgstr ""

#: module/database/FileDatabase.py:45
msgid "online"
msgstr ""

#: module/database/FileDatabase.py:45
msgid "queued"
msgstr "in wachtrij"

#: module/database/FileDatabase.py:45
msgid "skipped"
msgstr "overgeslagen"

#: module/database/FileDatabase.py:45
msgid "waiting"
msgstr "wachtend"

#: module/database/FileDatabase.py:45
msgid "temp. offline"
msgstr "tijdelijk offline"

#: module/database/FileDatabase.py:45
msgid "starting"
msgstr "startend"

#: module/database/FileDatabase.py:45
msgid "failed"
msgstr "mislukt"

#: module/database/FileDatabase.py:45
msgid "aborted"
msgstr "afgebroken"

#: module/database/FileDatabase.py:45
msgid "decrypting"
msgstr "decrypten"

#: module/database/FileDatabase.py:45
msgid "custom"
msgstr "gepersonaliseerd"

#: module/database/FileDatabase.py:45
msgid "downloading"
msgstr "wordt gedownload"

#: module/database/FileDatabase.py:45
msgid "processing"
msgstr "verwerken"

#: module/database/FileDatabase.py:45
msgid "unknown"
msgstr "onbekend"

#: module/database/FileDatabase.py:531 module/plugins/addon/IRCInterface.py:74
#: module/plugins/addon/XMPPInterface.py:83
#, python-format
msgid "Package finished: %s"
msgstr "Pakket compleet: %s"

#: module/remote/ThriftBackend.py:39
msgid "Using SSL ThriftBackend"
msgstr "SSL ThriftBackend gebruiken"

#: module/manager/RemoteManager.py:35
#, python-format
msgid "Remote backend error: %s"
msgstr "Remotebackendfout: %s"

#: module/manager/RemoteManager.py:82
#, python-format
msgid "Starting %(name)s: %(addr)s:%(port)s"
msgstr "Opstarten %(name)s: %(addr)s:%(port)s"

#: module/manager/RemoteManager.py:84
#, python-format
msgid "Failed loading backend %(name)s | %(error)s"
msgstr "Laden van backend %(name)s | %(error)s mislukt"

#: module/web/json_app.py:57
#, python-format
msgid "waiting %s"
msgstr "wachtend %s"

#: module/thread/ServerThread.py:35
msgid "SSL certificates not found."
msgstr "SSL certificaten niet gevonden."

#: module/thread/ServerThread.py:39
#, python-format
msgid "Sorry, we dropped support for starting %s directly within pyLoad"
msgstr "Sorry, we hebben ondersteuning voor starten van %s vanuit pyLoad verwijderd"

#: module/thread/ServerThread.py:40
msgid "You can use the threaded server which offers good performance and ssl,"
msgstr "Je kan de threaded server gebruiken welke goede prestaties en ssl biedt,"

#: module/thread/ServerThread.py:41
#, python-format
msgid "of course you can still use your existing %s with pyLoads fastcgi server"
msgstr "ook kan je de pyLoads fastcgi server gebruiken met je bestaande %s"

#: module/thread/ServerThread.py:42
msgid "sample configs are located in the module/web/servers directory"
msgstr "voorbeeldconfiguratie is te vinden in module/web/servers map"

#: module/thread/ServerThread.py:49
#, python-format
msgid "Can't use %(server)s, python-flup is not installed!"
msgstr "Kan %(server)s niet gebruiken, python-flup is niet geinstalleerd!"

#: module/thread/ServerThread.py:56
#, python-format
msgid "Error importing lightweight server: %s"
msgstr "Fout bij importeren lichtgewicht server: %s"

#: module/thread/ServerThread.py:57
msgid "You need to download and compile bjoern, https://github.com/jonashaag/bjoern"
msgstr "Je moet bjoern downloaden en compileren, https://github.com/jonashaag/bjoern"

#: module/thread/ServerThread.py:58
msgid "Copy the boern.so to module/lib folder or use Setup.py install"
msgstr "Kopieer de boern.so naar module/lib map en gebruik Setup.py install"

#: module/thread/ServerThread.py:59
msgid "Of course you need to be familiar with linux and know how to compile software"
msgstr "Hiervoor is kennis van linux en compileren van software een vereiste"

#: module/thread/ServerThread.py:63
msgid "Server set to threaded, due to known performance problems on windows."
msgstr "Server op threated gezet, vanwege bekende prestatie problemen op windows."

#: module/thread/ServerThread.py:80 module/thread/ServerThread.py:103
msgid "This server offers no SSL, please consider using threaded instead"
msgstr "Deze server heeft geen SSL, overweeg de threaded server te gebruiken"

#: module/thread/ServerThread.py:82
#, python-format
msgid "Starting builtin webserver: %(host)s:%(port)d"
msgstr "Opstarten van builltin webserver: %(host)s:%(port)d"

#: module/thread/ServerThread.py:87
#, python-format
msgid "Starting threaded SSL webserver: %(host)s:%(port)d"
msgstr "Opstarten van threaded SSL webserver: %(host)s:%(port)d"

#: module/thread/ServerThread.py:91
#, python-format
msgid "Starting threaded webserver: %(host)s:%(port)d"
msgstr "Opstarten van threaded webserver: %(host)s:%(port)d"

#: module/thread/ServerThread.py:97
#, python-format
msgid "Starting fastcgi server: %(host)s:%(port)d"
msgstr "Opstarten van fastcgi server: %(host)s:%(port)d"

#: module/thread/ServerThread.py:105
#, python-format
msgid "Starting lightweight webserver (bjoern): %(host)s:%(port)d"
msgstr "Opstarten van lichtgewicht webserver (bjoern): %(host)s:%(port)d"

#: module/web/pyload_app.py:125
msgid "You dont have permission to access this page."
msgstr "Je hebt geen rechten deze pagina te bezoeken."

#: module/web/pyload_app.py:193
msgid "Download directory not found."
msgstr "Downloadmap niet gevonden."

#: module/web/pyload_app.py:260 module/web/pyload_app.py:267
msgid "unlimited"
msgstr "onbeperkt"

#: module/web/pyload_app.py:262 module/web/pyload_app.py:269
msgid "not available"
msgstr "niet beschikbaar"

#: module/web/pyload_app.py:508
msgid "Run pyload.py -s to access the setup."
msgstr "Start pyload.py -s om setup te starten."

#: module/network/HTTPDownload.py:245
#, python-format
msgid "Download chunks failed, fallback to single connection | %s"
msgstr "Downloaden in chunks is mislukt, val terug op één connectie | %s"

#: module/thread/PluginThread.py:183
#, python-format
msgid "Download starts: %s"
msgstr "Download gestart: %s"

#: module/thread/PluginThread.py:189
#, python-format
msgid "Download finished: %s"
msgstr "Download compleet: %s"

#: module/thread/PluginThread.py:194 module/thread/PluginThread.py:365
#, python-format
msgid "Plugin %s is missing a function."
msgstr "Plugin %s mist een functie."

#: module/thread/PluginThread.py:202 module/thread/PluginThread.py:265
#: module/thread/PluginThread.py:382
#, python-format
msgid "Download aborted: %s"
msgstr "Download afgebroken: %s"

#: module/thread/PluginThread.py:222
#, python-format
msgid "Download restarted: %(name)s | %(msg)s"
msgstr "Download herstart: %(name)s | %(msg)s"

#: module/thread/PluginThread.py:231 module/thread/PluginThread.py:373
#, python-format
msgid "Download is offline: %s"
msgstr "%s offline"

#: module/thread/PluginThread.py:234
#, python-format
msgid "Download is temporary offline: %s"
msgstr "Download is tijdelijk offline: %s"

#: module/thread/PluginThread.py:237 module/thread/PluginThread.py:304
#, python-format
msgid "Download failed: %(name)s | %(msg)s"
msgstr "Download mislukt: %(name)s | %(msg)s"

#: module/thread/PluginThread.py:254
msgid "Couldn't connect to host or connection reset, waiting 1 minute and retry."
msgstr "Kan geen verbinding maken met host, wacht 1 minuut en probeer opnieuw."

#: module/thread/PluginThread.py:290
#, python-format
msgid "Download skipped: %(name)s due to %(plugin)s"
msgstr "Download overgeslagen: %(name)s vanwege %(plugin)s"

#: module/thread/PluginThread.py:361
#, python-format
msgid "Decrypting starts: %s"
msgstr "Decrypten gestart: %s"

#: module/thread/PluginThread.py:376 module/thread/PluginThread.py:394
#, python-format
msgid "Decrypting failed: %(name)s | %(msg)s"
msgstr "Decrypten mislukt: %(name)s | %(msg)s"

#: module/thread/PluginThread.py:388
#, python-format
msgid "Retrying %s"
msgstr "Opnieuw proberen %s"

#: module/thread/PluginThread.py:635
#, python-format
msgid "Info Fetching for %(name)s failed | %(err)s"
msgstr "Informatie verzamelen voor %(name)s mislukt | %(err)s"

#: module/manager/AddonManager.py:90 module/plugins/Addon.py:102
#, python-format
msgid "Error executing addon: %s"
msgstr "Uitvoeren van addon mislukt: %s"

#: module/manager/AddonManager.py:140
#, python-format
msgid "Failed activating %(name)s"
msgstr "Activeren mislukt van %(name)s"

#: module/manager/AddonManager.py:144
#, python-format
msgid "Activated plugins: %s"
msgstr "Geactiveerde plugins: %s"

#: module/manager/AddonManager.py:145
#, python-format
msgid "Deactivate plugins: %s"
msgstr "Niet geactiveerde plugins: %s"

#: module/manager/ThreadManager.py:137
#, python-format
msgid "Reconnect Failed: %s"
msgstr "Opnieuw verbinden mislukt: %s"

#: module/manager/ThreadManager.py:176
msgid "Reconnect script not found!"
msgstr "Opnieuw verbinden script niet gevonden!"

#: module/manager/ThreadManager.py:182
msgid "Starting reconnect"
msgstr "Opnieuw aan het verbinden"

#: module/manager/ThreadManager.py:196
msgid "Failed executing reconnect script!"
msgstr "Uitvoeren van opnieuw verbinden script mislukt!"

#: module/manager/ThreadManager.py:208
#, python-format
msgid "Reconnected, new IP: %s"
msgstr "Opnieuw verbonden, nieuwe IP: %s"

#: module/manager/ThreadManager.py:288
msgid "Not enough space left on device"
msgstr "Onvoldoende schijfruimte vrij"

#: module/plugins/Account.py:85 module/plugins/Account.py:91
#, python-format
msgid "Could not login with account %(user)s | %(msg)s"
msgstr "Kan niet inloggen onder gebruikersnaam %(user)s | %(msg)s"

#: module/plugins/Account.py:86
msgid "Wrong Password"
msgstr "Verkeerd wachtwoord opgegeven"

#: module/plugins/Account.py:240
#, python-format
msgid "Your Time %s has wrong format, use: 1:22-3:44"
msgstr "Tijd %s heeft verkeerde , gebruik: 1:22-3:44"

#: module/plugins/Account.py:266
#, python-format
msgid "Account %s has not enough traffic, checking again in 30min"
msgstr "Account %s heeft niet genoeg credits, wij proberen het opnieuw in 30min"

#: module/plugins/Account.py:273
#, python-format
msgid "Account %s is expired, checking again in 1h"
msgstr "Account %s is verlopen, opnieuw te controleren in 1h"

#: module/plugins/crypter/SerienjunkiesOrg.py:128
msgid "Downloadlimit reached"
msgstr "Downloadlimiet bereikt"

#: module/manager/PluginManager.py:153
#, python-format
msgid "%s has a invalid pattern."
msgstr "%s heeft een ongeldig patroon."

#: module/manager/PluginManager.py:272
#, python-format
msgid "Error importing %(name)s: %(msg)s"
msgstr "Fout bij importeren %(name)s: %(msg)s"

#: module/plugins/internal/MultiHoster.py:133
msgid "No Hoster loaded"
msgstr "Geen Hoster geladen"

#: module/plugins/account/BitshareCom.py:37
msgid "Activate direct Download in your Bitshare Account"
msgstr "Activeer Direct Downloaden in je Bitshare Account"

#: module/plugins/container/LinkList.py:65
msgid "LinkList could not be cleared."
msgstr "LinkList kon niet gewist worden."

#: module/manager/AccountManager.py:88
msgid "Account settings deleted, due to new config format."
msgstr "Accountgegevens verwijderd, vanwege nieuw configutatie formaat."

#: module/plugins/hoster/BasePlugin.py:65
msgid "Authorization required (username:password)"
msgstr "Autorisatie vereist (gebruikersnaam:wachtwoord)"

#: module/plugins/hoster/SimplydebridCom.py:25
#: module/plugins/hoster/RealdebridCom.py:41
#: module/plugins/hoster/FreeWayMe.py:38 module/plugins/hoster/ZeveraCom.py:22
#: module/plugins/hoster/UnrestrictLi.py:53
#: module/plugins/hoster/Premium4Me.py:28 module/plugins/hoster/FastixRu.py:36
#: module/plugins/hoster/AlldebridCom.py:38
#: module/plugins/hoster/DebridItaliaCom.py:40
#: module/plugins/hoster/RPNetBiz.py:26
#: module/plugins/hoster/MultiDebridCom.py:41
#: module/plugins/hoster/ReloadCc.py:24 module/plugins/hoster/RehostTo.py:26
#: module/plugins/hoster/PremiumizeMe.py:22
#: module/plugins/addon/RPNetBiz.py:43
#, python-format
msgid "Please enter your %s account or deactivate this plugin"
msgstr "Voer hier uw %s gegevens in of deactiveer deze plugin"

#: module/plugins/hoster/FilesMailRu.py:99
#, python-format
msgid "There was HTML Code in the Downloaded File (%s)...redirect error? The Download will be restarted."
msgstr "Tijdens het proberen van het ophalen van het volgende bestand (%s) is er een html/pagina reactie teruggekomen, dit kan een doorsturing zijn naar een andere pagina. pyLoad zal nu opnieuw het bestand proberen binnen te halen."

#: module/plugins/hoster/NetloadIn.py:146
#: module/plugins/hoster/NetloadIn.py:170
msgid "File temporarily not available"
msgstr "Bestand tijdelijk niet beschikbaar"

#: module/plugins/hoster/NetloadIn.py:183
#, python-format
msgid "Netload: waiting between downloads %d s."
msgstr "Netload: wachten tussen downloads %d s."

#: module/plugins/hoster/NetloadIn.py:214
#, python-format
msgid "Netload: waiting for captcha %d s."
msgstr "Netload: Wachten tot captcha %d s."

#: module/plugins/hoster/NetloadIn.py:252
msgid "Downloaded File was empty"
msgstr "Downloadlink heeft geen bestand gekoppeld"

#: module/plugins/hoster/UploadedTo.py:129
msgid "API key invalid"
msgstr "API sleutel ongeldig"

#: module/plugins/hoster/UploadedTo.py:153
#, python-format
msgid "%s: Not enough traffic left"
msgstr "%s: Overdracht limiet bijna bereikt , niet voldoende credits"

#: module/plugins/hoster/UploadedTo.py:156
msgid "Traffic exceeded"
msgstr "Dataverkeer overschreden"

#: module/plugins/hoster/RapidshareCom.py:100
msgid "Rapidshare: Traffic Share (direct download)"
msgstr "Rapidshare: Bandbreedte Delen (Direct Downloaden)"

#: module/plugins/hoster/RapidshareCom.py:127
#: module/plugins/hoster/RapidshareCom.py:194
msgid "Already downloading from this ip address, waiting 60 seconds"
msgstr "Er wordt al een bestand gedownload vanaf dit ip adres, na 60 seconden volgende poging"

#: module/plugins/hoster/RapidshareCom.py:131
msgid "Invalid Auth Code, download will be restarted"
msgstr "Ongeldige Auth Code, download wordt automatisch herstart"

#: module/plugins/hoster/RapidshareCom.py:199
msgid "RapidShareCom: No free slots"
msgstr "RapidShareCom: Geen beschikbaarde downloadslots"

#: module/plugins/hoster/RapidshareCom.py:202
msgid "You need a premium account for this file"
msgstr "U heeft een premium account nodig voor deze bestand"

#: module/plugins/hoster/RapidshareCom.py:204
msgid "Filename reported invalid"
msgstr "Bestandsnaam geeft ongeldigheidsmelding"

#: module/plugins/hoster/FileserveCom.py:99
msgid "Parallel download error, now waiting 60s."
msgstr "Gelijktijdige download mislukt, wacht 60 seconden voor volgende poging."

#: module/plugins/hoster/FileserveCom.py:215
msgid "Not logged in."
msgstr "Niet ingelogd."

#: module/plugins/hoster/MegaNz.py:56
msgid "Decryption failed"
msgstr "Ontcijferen van codering mislukt"

#: module/plugins/hoster/MegaNz.py:106
msgid "No file key provided in the URL"
msgstr "Geen bestandssleutel meegeleverd door URL"

#: module/plugins/hoster/MegaNz.py:118
msgid "Error code:"
msgstr "Foutmeldingscode:"

#: module/plugins/Container.py:68
msgid "File not exists."
msgstr "Bestand bestaat niet."

#: module/plugins/addon/UpdateManager.py:74
msgid "*** Plugins have been updated, please restart pyLoad ***"
msgstr "*** Plugins zijn bijgewerkt, pyLoad opnieuw starten aub***"

#: module/plugins/addon/UpdateManager.py:76
msgid "Plugins updated and reloaded"
msgstr "Plugins bijgewerkt en opnieuw geladen"

#: module/plugins/addon/UpdateManager.py:79
msgid "No plugin updates available"
msgstr "Geen plugin updates beschikbaar"

#: module/plugins/addon/UpdateManager.py:96
msgid "No Updates for pyLoad"
msgstr "Geen Updates voor pyLoad"

#: module/plugins/addon/UpdateManager.py:100
#, python-format
msgid "***  New pyLoad Version %s available  ***"
msgstr "***  Nieuwe pyLoad Versie %s beschikbaar  ***"

#: module/plugins/addon/UpdateManager.py:101
msgid "***  Get it here: http://pyload.org/download  ***"
msgstr "***  Download hier: http://pyload.org/download  ***"

#: module/plugins/addon/UpdateManager.py:104
msgid "Not able to connect server for updates"
msgstr "Niet in staat te verbinden met server voor updates"

#: module/plugins/addon/UpdateManager.py:148
#, python-format
msgid "New version of %(type)s|%(name)s : %(version).2f"
msgstr "Nieuwe versie van %(type)s|%(name)s : %(version).2f"

#: module/plugins/addon/UpdateManager.py:157
#: module/plugins/addon/UpdateManager.py:162
#, python-format
msgid "Error when updating %s"
msgstr "Fout tijdens updaten van %s"

#: module/plugins/addon/UpdateManager.py:162
msgid "Version mismatch"
msgstr "Versie mismatch"

#: module/plugins/addon/IRCInterface.py:82
#, python-format
msgid "Download finished: %(name)s @ %(plugin)s "
msgstr "Download compleet: %(name)s @ %(plugin)s "

#: module/plugins/addon/IRCInterface.py:95
#, python-format
msgid "New Captcha Request: %s"
msgstr "Nieuw Captcha verzoek: %s"

#: module/plugins/addon/IRCInterface.py:96
#, python-format
msgid "Answer with 'c %s text on the captcha'"
msgstr "Beantwoord met 'c%s text on the captcha'"

#: module/plugins/addon/Premium4Me.py:29
msgid "Please add your premium.to account first and restart pyLoad"
msgstr ""

#: module/plugins/addon/HotFolder.py:82
#, python-format
msgid "Added %s from HotFolder"
msgstr "Toegevoegde %s van HotFolder"

#: module/plugins/addon/ExtractArchive.py:95
#, python-format
msgid "No %s installed"
msgstr "Niet %s geinstalleerd"

#: module/plugins/addon/ExtractArchive.py:97
#: module/plugins/addon/ExtractArchive.py:102
#, python-format
msgid "Could not activate %s"
msgstr "Kan %s niet activeren"

#: module/plugins/addon/ExtractArchive.py:107
msgid "Activated"
msgstr "Geactiveerd"

#: module/plugins/addon/ExtractArchive.py:109
msgid "No Extract plugins activated"
msgstr "Geen uitpak plug-ins geactiveerd"

#: module/plugins/addon/ExtractArchive.py:121
#, python-format
msgid "Package %s queued for later extracting"
msgstr "Pakket %s in wachtrij voor later uitpakken"

#: module/plugins/addon/ExtractArchive.py:144
#, python-format
msgid "Check package %s"
msgstr "Controleer pakket %s"

#: module/plugins/addon/ExtractArchive.py:185
#, python-format
msgid "Extract to %s"
msgstr "Uitpakken naar %s"

#: module/plugins/addon/ExtractArchive.py:200
msgid "No files found to extract"
msgstr "Geen bestanden gevonden om uit te pakken"

#: module/plugins/addon/ExtractArchive.py:207
msgid "extracting"
msgstr "uitpakken"

#: module/plugins/addon/ExtractArchive.py:218
msgid "Password protected"
msgstr "Beveiligd met een wachtwoord"

#: module/plugins/addon/ExtractArchive.py:239
msgid "Wrong password"
msgstr "Verkeerd wachtwoord"

#: module/plugins/addon/ExtractArchive.py:247
#, python-format
msgid "Deleting %s files"
msgstr "%s bestanden verwijderen"

#: module/plugins/addon/ExtractArchive.py:254
msgid "Extracting finished"
msgstr "Uitpakken voltooid"

#: module/plugins/addon/ExtractArchive.py:260
msgid "Archive Error"
msgstr "Archief fout"

#: module/plugins/addon/ExtractArchive.py:262
msgid "CRC Mismatch"
msgstr "CRC foutief"

#: module/plugins/addon/ExtractArchive.py:266
msgid "Unknown Error"
msgstr "Onbekende fout"

#: module/plugins/addon/ExtractArchive.py:318
msgid "Setting User and Group failed"
msgstr "Instellen van gebruikers en de groep is mislukt"

#: module/plugins/addon/ClickAndLoad.py:75
msgid "Click'N'Load: Port 9666 already in use"
msgstr "Click'N'Load: Poort 9666 al in gebruik"

#: module/plugins/addon/CaptchaTrader.py:69
#: module/plugins/addon/Captcha9kw.py:59
#: module/plugins/addon/ExpertDecoders.py:50
#, python-format
msgid "%s credits left"
msgstr "%s credits over"

#: module/plugins/addon/CaptchaTrader.py:117
msgid "Could not send response."
msgstr "Kan het antwoord niet verzenden."

#: module/plugins/addon/CaptchaTrader.py:135
msgid "Your CaptchaTrader Account has not enough credits"
msgstr "Uw CaptchaTrader Account heeft niet genoeg credits"

#: module/plugins/addon/LinkdecrypterCom.py:43
msgid "Crypter list not found"
msgstr "Crypter lijst niet gevonden"

#: module/plugins/addon/LinkdecrypterCom.py:57
msgid "Crypter list is empty"
msgstr "Crypter lijst is leeg"

#: module/plugins/addon/XMPPInterface.py:91
#, python-format
msgid "Download finished: %(name)s @ %(plugin)s"
msgstr "Download compleet: %(name)s @ %(plugin)s"

#: module/plugins/addon/Captcha9kw.py:93
#, python-format
msgid "New CaptchaID from upload: %s : %s"
msgstr "Nieuwe CaptchaID van upload: %s: %s"

#: module/plugins/addon/Captcha9kw.py:129
msgid "Your Captcha 9kw.eu Account has not enough credits"
msgstr "Uw Captcha 9kw.eu Account heeft niet genoeg credits"

#: module/plugins/addon/ExternalScripts.py:54
#, python-format
msgid "Installed scripts for %s: "
msgstr "Geïnstalleerde scripts voor %s: "

#: module/plugins/addon/ExternalScripts.py:69
msgid "Script not executable:"
msgstr "Script niet uitvoerbaar:"

#: module/plugins/addon/ExternalScripts.py:80
#, python-format
msgid "Error in %(script)s: %(error)s"
msgstr "Fout in %(script)s: %(error)s"

#: module/plugins/addon/ExpertDecoders.py:95
msgid "Your ExpertDecoders Account has not enough credits"
msgstr "Uw ExpertDecoders Account heeft niet genoeg credits"

#: module/plugins/addon/RehostTo.py:32
msgid "Please add your rehost.to account first and restart pyLoad"
msgstr "Gelieve eerst uw premium account toe te voegen en daarna pyLoad te herstarten"

#: module/plugins/addon/PremiumizeMe.py:48
msgid "Please add a valid premiumize.me account first and restart pyLoad."
msgstr ""

#: module/plugins/addon/CaptchaBrotherhood.py:69
#, python-format
msgid "%d credits left"
msgstr "uw resterende aantal credits : %d"

#: module/plugins/Base.py:389
msgid "Pil and tesseract not installed and no Client connected for captcha decrypting"
msgstr "Captcha ontcijfering niet mogelijk : A. Client is niet verbonden & B. Pil en Tesseract module niet gevonden"

#: module/plugins/Base.py:393
msgid "No captcha result obtained in appropiate time by any of the plugins."
msgstr "Geen captcha resultaat ontvangen in aangegeven tijd bij een van de plugins."

#: module/plugins/Base.py:498 module/plugins/Base.py:532
#, python-format
msgid "Setting User and Group failed: %s"
msgstr "Instellingen van gebruiker en groep kunnen niet geladen worden : %s"

#: module/manager/CaptchaManager.py:78
msgid "No Client connected for captcha decrypting"
msgstr "Geen verbinding met client om captcha te decrypten"

#: module/Api.py:330
#, python-format
msgid "Added package %(name)s containing %(count)d links"
msgstr "Pakket %(name)s toevoegen met %(count)d links"

#: module/Api.py:593
#, python-format
msgid "Added %(count)d links to package #%(package)d "
msgstr "%(count)d links toevoegen aan pakket #%(package)d "

#: module/utils/JsEngine.py:156
msgid "No js engine detected, please install either Spidermonkey, ossp-js, pyv8 or rhino"
msgstr "Geen js engine gedetecteerd, installeer Spidermonkey, ossp-js, pyv8 of rhino"

